common/src/com/thoughtworks/go/domain/builder/Builder.java

```diff
@@ -101,23 +94,23 @@ public int hashCode() {
     }
 
     public void cancel(DefaultGoPublisher publisher, EnvironmentVariableContext environmentVariableContext, TaskExtension taskExtension) {
-        publisher.consumeLineWithPrefix("Start to execute cancel task: " + cancelBuilder.getDescription());
+        publisher.consumeLineWithPrefix("Cancel task: " + cancelBuilder.getDescription());
```

Comment:
Sorry for the late reply - I've since changed it to read "On Cancel Task: ...", which is the same wording in UI. Does that work?